AccessCheckoutSDK/Podfile.lock

```diff
@@ -29,27 +21,19 @@ DEPENDENCIES:
 
 SPEC REPOS:
   trunk:
-    - Alamofire
-    - BrightFutures
     - Cuckoo
     - Mockingjay
     - Nimble
     - PactConsumerSwift
-    - Quick
-    - Result
     - URITemplate
 
 SPEC CHECKSUMS:
-  Alamofire: ae5c501addb7afdbb13687d7f2f722c78734c2d3
-  BrightFutures: 982d41952f770d6a9d8335f5f1d8b9c81a6ff430
   Cuckoo: 57f71c6616b6b08cd9fe27f803da056fb619ff3e
   Mockingjay: 0f7c5aa49c7f1b95621cee3c79b557141f5a225c
-  Nimble: 622629381bda1dd5678162f21f1368cec7cbba60
-  PactConsumerSwift: 64078e5589fa970d67b66bb5bd9b654c22a7f036
-  Quick: 4be43f6634acfa727dd106bdf3929ce125ffa79d
-  Result: bd966fac789cc6c1563440b348ab2598cc24d5c7
+  Nimble: 98b888285a615fd34f20e61753cf58ea1402bde4
+  PactConsumerSwift: b7eafe230643a309f65f5e0bbda1f21d068cc8aa
   URITemplate: 58e0d47f967006c5d59888af5356c4a8ed3b197d
 
 PODFILE CHECKSUM: 533bc3de53ecdc768b831ced07ef125060d68a8f
 
-COCOAPODS: 1.8.4
+COCOAPODS: 1.9.0
```

Comment:
The update to Cocoapods 1.9 should be a different commit. The reason for that is if we were to have issues with that version then we would just have to revert that small commit against reverting this whole commit.